sockeye/arguments.py

```diff
@@ -960,6 +964,13 @@ def add_translate_cli_args(params):
     add_logging_args(params)
 
 
+def add_max_output_cli_args(params):
+    params.add_argument('--max-output-length',
+                        type=int,
+                        default=None,
+                        help='Number of words to generate during translation. It None, it will be computed automatically.')
```

Comment:
typo: It -> If.
Please add "Default: %(default)s."

Reply:
Thanks. I will double-check if all the new params have "Default: %(default)s.".